components/partnerspage/componentpartners.htm

```diff
@@ -1,7 +1,12 @@
 <div class="list">
+{% if template == 'template2' %}
+	{% for partner_image in partners %}
+		<img src="{{partner_image.cover.getThumb(210, 60, 'fit')}}" style="margin: 0 10px 0 10px;">
+	{% endfor %}
+{% endif%}
 {% for partner in partners %}
         <div class="list-item align-items-start">
-            {%if partner.cover %}
+            {%if partner.cover and template != 'template2' %}

```

Comment:
{%if partner.cover and template != 'template2' %}
to be
{%if partner.cover and not is_hidden_cover %}